ecs_fargate/README.md

```diff
@@ -102,7 +102,7 @@ Resources:
               enable-ecs-log-metadata: true
           MemoryReservation: 50
 ```
-
+**Note**: You can also use a [TaskDefinition secret][37] if you do not want to expose the apiKey as a plain-text variable.
```

Comment:
```suggestion
**Note**: Use a [TaskDefinition secret][37] to avoid exposing the `apikey` in plain text.

```
Need a new line or else Markdown sticks the next sentence into the same line as the Note.